usaspending_api/awards/delta_models/financial_accounts_by_awards.py

```diff
@@ -80,7 +80,7 @@
             UPPER(faba.fain) AS fain,
             UPPER(faba.uri) AS uri
         FROM
-            int.financial_accounts_by_awards AS faba
+            raw.financial_accounts_by_awards AS faba
```

Comment:
Had to change these back to `raw` because the `int` schema already has all the changes (and shows no records pending updates) by the time we write the final temp view to Postgres